ts-rs/src/export.rs

```diff
@@ -221,6 +222,14 @@ fn import_path(from: &Path, import: &Path) -> String {
     }
 }
 
+fn to_absolute_path(path: &Path) -> PathBuf {
+    let Ok(current_path) = std::env::current_dir() else {
+        return path.to_owned()
+    };
+
+    current_path.join(path).clean()
```

Comment:
I think we should just copy [these 16](https://docs.rs/path-clean/latest/src/path_clean/lib.rs.html#62-78) lines into here.

Reply:
Fair enough, that avoids a dependency

Reply:
The `diff_paths` function has a comment about licensing. Should I copy it over to this as well, `path-clean` uses the same license described there (APACHE or MIT)

Reply:
Alright, I did end up adding the licensing info. But I'm also starting to think the `export.rs` file is doing a lot more than it should. Do you mind if I break some of it into other modules?